tests/integration/test_model/test_model_recipe.py

```diff
@@ -146,12 +146,11 @@ def test_custom_args(self):
             job.algorithm_spec.code_dir.location_value.bucket,
             session.oss_bucket.bucket_name,
         )
-    
+
     def test_compression(self):
         model = RegisteredModel(model_name="qwen2-0.5b-instruct", model_provider="pai")
         compress_recipe = model.model_recipe(
-            recipe_type=ModelRecipeType.COMPRESSION,
-            method='Quantization:MinMax-8Bit'
+            recipe_type=ModelRecipeType.COMPRESSION, method="Quantization:MinMax-8Bit"
```

Comment:
`method='Quantization:MinMax-8Bit',`
add a comma to avoid auto fix?

Reply:
The previous commit resulted in a failure of the lint test. Therefore, we should address the lint issues first.

I have no preference between the code styles.